test/auth.test.ts

```diff
@@ -0,0 +1,46 @@
+import AES from "crypto-js/aes";
+
+import { setupRecording, setupAPI } from "../test/setup";
+import { SaleorSDK } from "../src/core";
+
+describe("auth api", () => {
+  console.log(process.env);
+  const context = setupRecording();
+  const { client } = setupAPI();
+  const saleor = SaleorSDK(client);
+
+  beforeEach(() => {
+    const { server } = context.polly;
+    server.any().on("beforePersist", (_, recording) => {
+      recording.request = AES.encrypt(
+        recording.request,
+        process.env.SECRET_KEY
+      );
+      recording.response = AES.encrypt(
+        recording.response,
+        process.env.SECRET_KEY
+      );
+    });
+  });
+
+  it("can login", async () => {
+    const { data } = await saleor.auth.login("sdk@example.com", "test1");
```

Comment:
Use env vars for auth credentials: `TEST_AUTH_EMAIL` `TEST_AUTH_PASSWORD`

Reply:
done